FundTrack.DAL/Concrete/BankRepositoty.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using FundTrack.DAL.Abstract;
+using FundTrack.DAL.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace FundTrack.DAL.Concrete
+{
+    public class BankRepositoty : IBankRepository
+    {
+        private readonly FundTrackContext _context;
+
+        public BankRepositoty(FundTrackContext context)
+        {
+            _context = context;
+        }
+
+        public IEnumerable<Bank> Read()
+        {
+            return _context.Banks;
+        }
+
+        public Bank Get(int bankId)
+        {
+            return _context.Banks
+                .FirstOrDefault(b => b.Id == bankId);
+        }
+
+        public Bank Create(Bank item)
+        {
+            return _context.Banks.Add(item).Entity;
+        }
+
+        public Bank Update(Bank bank)
```

Comment:
in this method you don't  write  data to database.
use context.Banks.Update method